addons/mail/static/src/rtc/voice_analyser.js

```diff
@@ -0,0 +1,117 @@
+/* @odoo-module */
+import { Component, useState, useRef, onMounted } from "@odoo/owl";
+
+export class VoiceAnalyser extends Component {
```

Comment:
This is missing the doc string that type the component props.
If you want an example, you can look at the link preview component.